media/js/firefox/hello.js

```diff
@@ -0,0 +1,174 @@
+    var $videoContainer = $('#video-modal');
+    var $video = $('#hello-video');
+
+    var supportsHTML5Video = !!document.createElement('video').canPlayType;
+    var isWideViewport = $(window).width() >= 740;
+    var mqIsWide;
+    var isIE = /MSIE/.test(navigator.userAgent);
+    var isTrident = /Trident/.test(navigator.userAgent);
+    var isOldOpera= /Presto/.test(navigator.userAgent);
+
+    var supportsSVGAnimation = function() {
+        return supportsInlineSVG() && !isIE && !isTrident && !isOldOpera;
+    };
+
+    var supportsInlineSVG = function() {
+        var div = document.createElement('div');
+        div.innerHTML = '<svg/>';
+        return (div.firstChild && div.firstChild.namespaceURI) === 'http://www.w3.org/2000/svg';
+    };
+
+    if (isWideViewport) {
+        if (supportsSVGAnimation()) {
+            $(w).on('load', function() {
+                $animationStage.addClass('animate wide');
+            });
+        } else {
+            $('body').addClass('no-animation');
+        }
+    } else {
+        if (Modernizr.cssanimations) {
+            $animationStage.addClass('animate mini');
+        } else {
+            $('body').addClass('no-animation');
+        }
+    }
+
+    // resizing the browser with animation just displays the intro image
+    if (typeof matchMedia !== 'undefined') {
+        mqIsWide = matchMedia('(min-width: 760px)');
+
+        mqIsWide.addListener(function() {
+            if ($animationStage.hasClass('animate')) {
+                $animationStage.remove(); // why not?
+                $introImage.css('display', 'block');
+            }
+        });
+    }
+
+    // listen for events in/on Hello menu
+    var bindHelloObserver = function() {
+        Mozilla.UITour.observe(function(e) {
+            switch (e) {
+                case 'Loop:ChatWindowOpened':
+                    w.gaTrack(['_trackEvent', '/hello interactions', 'productPage', 'StartConversation-NoTour']);
+                    break;
+                case 'Loop:RoomURLCopied':
+                    w.gaTrack(['_trackEvent', '/hello interactions', 'productPage', 'URLCopied-NoTour']);
+                    break;
+                case 'Loop:RoomURLEmailed':
+                    w.gaTrack(['_trackEvent', '/hello interactions', 'productPage', 'URLEmailed-NoTour']);
+                    break;
+            }
+        });
+    };
+
+    var handleVisibilityChange = function() {
+        if (document.hidden) {
+            // hide Hello menu & stop observer when changing tabs or minimizing window
+            Mozilla.UITour.observe(null);
+        } else {
+            // listen for Hello menu/chat window events
+            bindHelloObserver();
+        }
+    };
+
+    if (w.isFirefox()) {
+        // if Fx, hide all footer messaging
+        // (correct messaging to display determined below)
+        $('.dltry-copy').hide();
+
+        // Hello exists in version 35 and up
+        if (w.getFirefoxMasterVersion() >= 35) {
```

Comment:
Ah, good point. I did a little testing with device spoofing. I agree showing the "Try Hello" button with the default SUMO link behavior is the best we can do for Android. I will add a check.